Bottom: fe1a90c2df4f142a4677092dd630d27a5c30fa3b
Top:    a410e93dce14d46dba90967a17ac7834c8984c5a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 02:01:23 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 6edd29d..4724169 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2623,7 +2623,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
